core/src/test/scala/tech/beshu/ror/unit/acl/factory/decoders/definitions/LdapServicesSettingsTests.scala

```diff
@@ -82,6 +82,39 @@ class LdapServicesSettingsTests(ldapConnectionPoolProvider: UnboundidLdapConnect
           }
         )
       }
+      "one LDAP service with circuit breaker is declared" in {
```

Comment:
missing tests for:
* default CB in no explicit CB config declared
* malformed CB section

_(We can do it in a second part of the task)_